js/index.js

```diff
@@ -1,6 +1,77 @@
-const startApp = () => {
-  const body = document.querySelector('body');
-  body.innerHTML = '<h2>JAVASCRIPT ENABLED</h2>';
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const contentDiv = document.querySelector('.content-div');
+const form = document.querySelector('.form');
```

Comment:
Since we are only trying to target one element here, It would make sense to use `ID` rather than classes